--- old
+++ new
@@ -1,0 +1,5 @@
+---
+layout: post
+title: "Thứ 4 18/3 ngày làm việc đầu tiên trên tàu"
+date: 2026-03-18
+---
@@ -6,1 +11,1 @@
-Buổi chiều bác Quí phân mỗi vị trí cử vài người ở lại để khám cho những người chưa khám, những người còn lại lên tham quan đảo. T ở lại phụ a Tuấn. Phòng ngoại da liễu bay ra đảo hết không để ai ở lại, vậy là bác Hùng bên quân y phải ra khám.  
+Buổi chiều bác Quí phân mỗi vị trí cử vài người ở lại để khám cho những người chưa khám, những người còn lại lên tham quan đảo. T ở lại phụ a T. Phòng ngoại da liễu bay ra đảo hết không để ai ở lại, vậy là bác Hùng bên quân y phải ra khám.  
@@ -9,1 +14,1 @@
-Bữa cơm chiều nhà bếp có 2 món cá, cá thu rim, cá gì nấu canh chua. Đã ặn được nhiều cá hơn. Ăn 1 bát cơm, còn lại toàn ăn đồ ăn hehe.
+Bữa cơm chiều nhà bếp có 2 món cá, cá thu rim, cá gì nấu canh chua. Đã ăn được nhiều cá hơn. Ăn 1 bát cơm, còn lại toàn ăn đồ ăn hehe.