--- old
+++ new
@@ -1,0 +1,5 @@
+---
+layout: post
+title: "Thứ 3 17/3 đảo Đá Lát"
+date: 2026-03-17
+---
@@ -9,1 +14,1 @@
-Ăn gala xong đi dạo trên cầu cảng. Được bác Quý dẫn cả đám đi thăm trưởng đảo bằng xe điện. Đi một quãng đến ngôi nhà 2 lầu 1 trệt gặp a Liêm trưởng đảo. Bác Quí, a Liêm nói chuyện với nhau, bọn trẻ trẻ chúng tôi nghe ké. Khi về tí bỏ quên ông B. Trên đảo nuôi khoảng chục con chó. Trên đường về có con cắn con gà chọi của một a lính.
+Ăn gala xong đi dạo trên cầu cảng. Được bác Quý dẫn cả đám đi thăm trưởng đảo bằng xe điện. Đi một quãng đến ngôi nhà 2 lầu 1 trệt gặp a Liêm trưởng đảo. Anh Liêm, giống như các anh lính trên đảo, da sạm đen vì nắng. Anh mời chúng tôi trà, bánh ngọt.Bác Quí, a Liêm nói chuyện với nhau, bọn trẻ trẻ chúng tôi nghe ké. Khi về tí bỏ quên ông B. Trên đảo nuôi khoảng chục con chó, con nào cũng béo tốt. Trên đường về có con cắn con gà chọi của một a lính.