--- old
+++ new
@@ -1,1 +1,1 @@
-Đêm liên hoan ngày làm việc cuối cùng tại đảo Thuyền Chài D. Tàu nhổ neo rời đảo ngay trong đêm. Chúng tôi trên boong tàu bật đèn flash điện thoại vẫy chào những người ở lại. Thuyền rời xa dần, tạm biệt hòn đảoCảm xúc.
+Đêm liên hoan ngày làm việc cuối cùng tại đảo Thuyền Chài D. Tàu nhổ neo rời đảo ngay trong đêm. Chúng tôi trên boong tàu bật đèn flash điện thoại vẫy chào những người ở lại. Thuyền rời xa dần, tạm biệt hòn đảo, giờ được thắp sáng bởi những ánh đèn sáng tròn nhỏ. Cảm xúc.