--- old
+++ new
@@ -1,0 +1,5 @@
+---
+layout: post
+title: "Chủ Nhật đảo An Bang"
+date: 2026-03-22
+---
@@ -2,0 +7,1 @@
+Buổi trưa sau thông báo gì đó của thuyền trưởng trên cabin, mọi người ùa ra mạn phải tàu, xa xa là những cá heo, có con nhảy ưỡn thân mình lên trên mặt bước trước khi lặn xuống.
@@ -5,1 +11,1 @@
-Trước khi đi có cảm, hắt xì nhẹ. Sau khi đi đã hết.
+Trước khi đi có cảm, hắt xì nhẹ. Sau khi đi đã hết. 