--- old
+++ new
@@ -1,2 +1,6 @@
-Đảo Thuyền Chài B
-Khoảng 7h30 8h thuyền nhổ neo chuyển qua đảo Thuyền Chài C
+---
+layout: post
+title: "Thứ 2 23/3 cụm đảo Thuyền Chài"
+date: 2026-03-23
+---
+Sau khi hoàn thành công việc tại Đảo Thuyền Chài B, khoảng 7h30-8h thuyền nhổ neo chuyển qua đảo Thuyền Chài C.
@@ -5,1 +9,2 @@
-Tàu đỗ sát đảo, ăn tối xong, t đi qua chiếc thang lên đảo đi dạo.
+Tàu đỗ sát đảo, ăn tối xong, mọi người đi qua chiếc thang lên đảo đi dạo.
+