--- old
+++ new
@@ -4,1 +4,1 @@
-<a href="https://ibb.co/0p35h6Cm"><img src="https://i.ibb.co/HLQZnbtB/20260315-151728.jpg" alt="20260315-151728" border="0"></a><br /><a target='_blank' href='https://imgbb.com/'>get html for blogs</a><br />
+<a href="https://ibb.co/0p35h6Cm"><img src="https://i.ibb.co/HLQZnbtB/20260315-151728.jpg" alt="20260315-151728" border="0"></a><br /><a target='_blank' href='https://imgbb.com/'>get html for blogs</a><br 
@@ -6,3 +6,0 @@
-[url=https://ibb.co/0p35h6Cm][img]https://i.ibb.co/HLQZnbtB/20260315-151728.jpg[/img][/url]
-[url=https://imgbb.com/]get html for blogs[/url]
-