--- old
+++ new
@@ -2,1 +2,0 @@
-https://maps.app.goo.gl/TEbtzMQ8LHuDxeh29?g_st=ac
@@ -4,1 +3,3 @@
-<a href="https://ibb.co/0p35h6Cm"><img src="https://i.ibb.co/0p35h6Cm/20260315-151728.jpg" 
+<a href="https://ibb.co/0p35h6Cm"><img src="https://i.ibb.co/HLQZnbtB/20260315-151728.jpg" alt="20260315-151728" border="0"></a><br /><a target='_blank' href='https://imgbb.com/'>get html for blogs</a><br />
+
+