--- old
+++ new
@@ -9,2 +9,1 @@
-Ăn gala xong đi dạo trên cầu cảng. Được bác Quý dẫn cả đám đi thăm trưởng đảo bằng xe điện. Đi một quãng đến ngôi nhà 2 lầu 1 trệt gặp a Liêm trưởng đảo. Bác Quí, a Liêm nói chuyện với nhau, bọn trẻ trẻ chúng tôi nghe ké. 
-Trên đảo nuôi khoảng chục con chó. Trên đường về có con cắn con gà chọi của một a lính.
+Ăn gala xong đi dạo trên cầu cảng. Được bác Quý dẫn cả đám đi thăm trưởng đảo bằng xe điện. Đi một quãng đến ngôi nhà 2 lầu 1 trệt gặp a Liêm trưởng đảo. Bác Quí, a Liêm nói chuyện với nhau, bọn trẻ trẻ chúng tôi nghe ké. Khi về tí bỏ quên ông B. Trên đảo nuôi khoảng chục con chó. Trên đường về có con cắn con gà chọi của một a lính.