--- old
+++ new
@@ -7,1 +7,1 @@
-12h58p, tàu đã tới gần đảo Đá Lát, có vẻ tàu thả neo ở đây không di chuyển nữa. Hòn đảo nhìn từ xa trắng xóa, có vài ngôi nhà mái đỏ, hầu như không có màu xanh của cây cối, cạnh bên là ngọn hải đăng.
+12h58p, tàu đã tới gần đảo Đá Lát, có vẻ tàu thả neo ở đây không di chuyển nữa. Nhìn từ xa bên mạn trái con tàu, hòn đảo trông trắng xóa, có vài ngôi nhà mái đỏ, hầu như không có màu xanh của cây cối, cạnh bên là ngọn hải đăng.