--- old
+++ new
@@ -6,2 +6,2 @@
-Buổi tổng kết được tổ chức tại bv Lê Văn Thịnh. Lần lượt từng người chúng tôi được gọi tên lên bục để trao bằng khen, huy hiệu chiến sĩ trường sa. Huy hiệu thật đẹp.
-Tạm biệt mọi người. 
+Buổi tổng kết được tổ chức tại bv Lê Văn Thịnh. Lần lượt từng người chúng tôi được gọi tên lên bục để trao bằng khen, huy hiệu chiến sĩ trường sa. Huy hiệu thật đẹp. 
+Tạm biệt mọi người...