--- old
+++ new
@@ -1,1 +1,6 @@
-Nhóm đi Trường Sa họp mặt ăn tối tại quán The Street trên đường Phan Đăng Lưu. Nhóm thiếu 2 người, a Phúc, a Tuấn bận việc nên không đi được. Quán có rượu mơ, rượu táo mèo. Rượu táo mèo khó uống hơn. Chúng tôi cùng hô lại những khẩu hiệu trong những bữa tiệc lúc còn ở Trường Sa. Trong toilet quán có chậu rửa tay đặt thấp, dùng để hòoo.
+---
+layout: post
+title: "Họp mắt nhóm Trường Sa"
+date: 2026-04-11
+---
+Nhóm đi Trường Sa họp mặt ăn tối tại quán The Street trên đường Phan Đăng Lưu. Nhóm thiếu 2 người, a Ph, a T bận việc nên không đi được. Quán có rượu mơ, rượu táo mèo. Rượu táo mèo khó uống hơn. Chúng tôi cùng hô lại những khẩu hiệu trong những bữa tiệc lúc còn ở Trường Sa. T tự hỏi có khi nào trong quán biết những khẩu hiệu này, tới mời bàn chúng tôi một ly. Trong toilet quán có chậu rửa tay đặt thấp, dùng để hòoo.