--- old
+++ new
@@ -1,1 +1,6 @@
-Đã đọc xong cuốn atomic habits - những thói quen nguyên tử. Dịch giả là cô Vũ Phi Yên, giảng viên bộ môn shtb hồi học ĐH.
+---
+layout: post
+title: "Thứ 6 27/3 ngày 5 trên đảo"
+date: 2026-03-27
+---
+Đã đọc xong cuốn atomic habits - những thói quen nguyên tử, sách của tk thủ trưởng. Dịch giả là cô Vũ Phi Yên, giảng viên bộ môn shtb hồi học ĐH. Ấn tượng nhất là câu mỗi hành động dù nhỏ, là một lá phiếu bầu quyết định xem bạn là người thế nào, đại loại vậy.
@@ -3,0 +8,1 @@
+