--- old
+++ new
@@ -1,0 +1,5 @@
+---
+layout: post
+title: "Thứ 5 19/3 đảo Đá Tây"
+date: 2026-03-19
+---
@@ -3,1 +8,1 @@
-Khoảng 11h, mặc áo phao và được thủy thủ đoàn hỗ trợ bước lên cano đi vào trong đảo tham quan. 
+Khoảng 11h, mặc áo phao và được thủy thủ đoàn hỗ trợ bước lên cano đi vào trong đảo tham quan. Chúng tôi tham quan Nhà đại đoàn kết, phía trước toà nhà to lớn này là cột mốc chủ quyền. Trên đảo có chùa Đá Tây A.
@@ -7,0 +12,1 @@
+