--- old
+++ new
@@ -2,1 +2,4 @@
-Chiều lên thuyền nhỏ vào thăm đảo An Bang. Sóng dập dềnh làm con thuyền dù đã cập sát mạn tàu nhưng vẫn dao động lên xuống nhiều. Chưa bao giờ lên xuống thuyền mà hồi hộp như này. Tim đập thình thịch.  
+Chiều lên thuyền nhỏ vào thăm đảo An Bang. Sóng dập dềnh làm con thuyền dù đã cập sát mạn tàu nhưng vẫn dao động lên xuống nhiều. Chưa bao giờ lên xuống thuyền mà hồi hộp như này. Tim đập thình thịch. Nhờ sự hỗ trợ của các anh thủy thủ đoàn mà lên tàu an toàn. Di chuyển trên tàu nhỏ sóng đánh lên xuống còn hơn đi bằng cano các đợt trước. Tàu cập đảo An Bàng, bước qua xà lan trên đó chở cát và có một máy xúc, rồi bước vào bờ. 
+Các thành viên trên đảo đứng theo hàng, lần lượt bắt tay chào các thành viên đoàn công tác. Chụp hình cùng cột mốc với các thành viên trên đảo. Tham quan trạm xá, trò chuyện cùng bs quân y trên đảo. 
+Lượt về sóng lớn không bước lên thuyền được nên chỉ huy quyết định nhờ máy xúc trên chiếc xà lan, xúc một lượt 4 người từ xà lan qua chiếc thuyền nhỏ để trở về tàu. Trải nghiệm lần đầu tiên ngồi trong cái xúc để được đưa từ bên này sang bên kia. 
+Trước khi đi có cảm, hắt xì nhẹ. Sau khi đi đã hết.