--- old
+++ new
@@ -1,1 +1,1 @@
-Sau một đêm dập dềnh, sáng dậy tàu đã thả neo. Phía xa là đảo An Bang. Sáng nay sóng lớn, cano chở đội xét nghiệm không vào đảo được nên phải đợi thuyền lớn. Mặc dù vậy, sóng vẫn to, thuyền lớn cập sát mạn trái phía đuôi tàu
+Sau một đêm dập dềnh, sáng dậy tàu đã thả neo. Phía xa là đảo An Bang. Sáng nay sóng lớn, cano chở đội xét nghiệm không vào đảo được nên phải đợi thuyền lớn. Mặc dù vậy, sóng vẫn to, thuyền lớn cập sát mạn trái phía đuôi tàu nhưng vẫn nhấp nhô, không thuận lợi để lên thuyền. Thuyền trưởng cho tàu nhổ neo, tìm vị trí sóng êm hơn. 