--- old
+++ new
@@ -1,1 +1,7 @@
-Buổi chiều, xe tải chở một vòng tham quan đảo Thuyền Chài D, đảo lớn nhất trong cụm đảo Thuyền Chài. Thắp nén hương trong đền thờ Bác Hồ, tham quan Khuông Việt Thiền Tự. Đi trên đường bay, nghe đảo trưởng bảo đạt chuẩn E4, máy bay boeing 747, airbus 350 có thể hạ cánh. Trên đảo có sân pickleball nữa.
+---
+layout: post
+title: "Thứ 3 24/3 đảo Thuyền Chài D ngày 2"
+date: 2026-03-24
+---
+Buổi chiều, xe tải chở một vòng tham quan đảo Thuyền Chài D, đảo lớn nhất trong cụm đảo Thuyền Chài. Thắp nén hương trong đền thờ Bác Hồ, tham quan Khuông Việt Thiền Tự. Đi trên đường bay, nghe đảo trưởng bảo đạt chuẩn E4, các máy bay lớn như boeing 747, airbus 350 có thể hạ cánh. Trên đảo có sân pickleball, cầu lông, bóng bàn nữa.
+