--- old
+++ new
@@ -6,1 +6,1 @@
-Buổi chiều bác Quí phân mỗi vị trí cử vài người ở lại để khám cho những người chưa khám, những người còn lại lên tham quan đảo. T ở lại phụ a Tuấn. Phòng ngoại da liễu bay ra đảo hết không để ai ở lại, vậy là bác gì bên quân y phải ra khám.  
+Buổi chiều bác Quí phân mỗi vị trí cử vài người ở lại để khám cho những người chưa khám, những người còn lại lên tham quan đảo. T ở lại phụ a Tuấn. Phòng ngoại da liễu bay ra đảo hết không để ai ở lại, vậy là bác Hùng bên quân y phải ra khám.  