--- old
+++ new
@@ -1,1 +1,3 @@
-Khoảng 20h, tiếng xích nặng nề được thả xuống, xun
+Khoảng 20h, tiếng xích nặng nề được thả xuống, xung quanh biển tối mịt. Phía xa là các ánh đèn nho nhỏ của các ngôi nhà trên đảo đá tây.
+Phía đuôi thuyền, mọi người đổ ra xem các thủy thủ câu cá. Họ cầm vợt đứng trên bục ra sát mạn tàu thả câu. Cần cong lên, dây cước căng ra, rồi kéo lên con cá thân dài nghe bảo là cá nhồng. Trong xô khoảng chục con giống vậy. Bên trái mạn tàu chiếu đèn rọi xuống biển, ông nhà bếp dùng cây vợt dài khoảng 7 8m thỉnh thoảng vợt được mấy con cá chuồn be bé.
+Khoảng 9h, tàu thông báo buổi ăn khuya. Món chè bí đỏ ban đầu trông lõng bõng nước mà ăn ngon phết.