--- old
+++ new
@@ -1,1 +1,4 @@
-
+Thức dậy trước khi loa thông báo để lên boong ngắm bình minh. Có thể sẽ rất lâu để có trải nghiệm này lần nữa. Ăn sáng xong về phòng, lau sơ lại sàn theo sự phân công của nhóm. Thay áo, mặc quần tây, khóa vali. Ra boong đã thấy tàu cập cảng. Các chiến sĩ dưới cảng và trên các tàu bên cạnh đã xếp hàng thành hàng ngang chào đón tàu trở về. Ấn tượng như ban đầu, cảng to lớn, nhiều thuyền lớn. Được sự hỗ trợ của các thuyền viên chuyền từng vali xuống. Bắt tay mọi người rồi lên xe. Chúng tôi dừng lại ăn trưa ở quán ăn hải sản, ăn uống cùng nhau, đến khoảng 11h rồi lên xe.
+Khoảng 16h30 về đến nhà. Tối ghi lại những dòng này, bên cửa sổ vang lên tiếng cười đùa của những đứa trẻ con hàng xóm. Trở về cuộc sống như mọi khi.
+Một trải nghiệm không thể nào quên. Một hành trình thật nhiều ý nghĩa. Nhiều điều cần học hỏi, rút kinh nghiệm. 
+Trường Sa 2026.